plugins/arclite/arclite_bel.hlf

```diff
@@ -0,0 +1,270 @@
+﻿.Language=Belarussian,Belarussian (Беларуская)
```

Comment:
When you add new files, make sure that `.Language` is spelled exactly the same as in the [main template](https://github.com/FarGroup/FarManager/blob/cbd4fcf1f338f0e85b65148d45d970e34d5182c5/far/farlang.templ.m4#L23), i.e. with one `s` in this case, otherwise it won't be picked.